viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -47,24 +47,34 @@ namespace viam {
 
 static const int checkForShutdownIntervalMicroseconds = 1e5;
 
-// This RGB value represents "no input" from a Cario pained map
-static const unsigned char defaultCairosEmptyPaintedSlice = 102;
 static const int jpegQuality = 50;
 // Byte limit on GRPC, used to help determine sampling skip_count
 static const int maximumGRPCByteLimit = 32 * 1024 * 1024;
 // Byte limit for chunks on GRPC, used for streaming apis
-static const int maximumGRPCByteChunkSize = 64 * 1024;
-// Coeffient to adjust the skip count for the PCD to ensure the file is within
-// grpc limitations. Increase the value if you expect dense feature-rich maps
-static const int samplingFactor = 1;
-// Conversion to number of bytes used in colored PCD encoding
-static const int pixelBytetoPCDByte = 16 / 4;
+static const int maximumGRPCByteChunkSize = 1 * 1024 * 1024;
+
 // Quaternion to rotate axes to the XZ plane
 static const Eigen::Quaterniond pcdRotation(0.7071068, -0.7071068, 0, 0);
 // Static offset quaternion, so orientation matches physical intuition.
 // This will result in rotations occurring within the y axis to match 2D mapping
 // in the XZ plane
 static const Eigen::Quaterniond pcdOffsetRotation(0.7071068, 0.7071068, 0, 0);
+
+// resolution defines the area in meters that each pixel represent. This
+// is used to draw the jpeg map and in so doing defines the resolution of
+// the outputted PCD
```

Comment:
[nit] Uppercase to match the style of the rest of the document
```suggestion
// Resolution defines the area in meters that each pixel represents. This
// is used to draw the jpeg map and, in so doing, defines the resolution of
// the outputted PCD
```

Reply:
I want to keep resolution lowercase to make it clear we are talking about the variable, however, I will add a word in from to be capitalized

Reply:
I see - that wasn't clear to me. Yeah, maybe `The resolution variable defines...` would have been less confusing for me.